src/lib/router.ts

```diff
@@ -0,0 +1,30 @@
+import WeakMap from '@dojo/shim/WeakMap';
+import { Context, Parameters, RouteInterface, RouterInterface } from '../interfaces';
+
+export const parentMap = new WeakMap<RouteInterface<Context, Parameters>, RouterInterface<Context>>();
```

Comment:
I went back and forth on this. This is needed by `Router` only to add to the map. It was either export this and let `Router` use it, or export a function that just calls `parentMap.set`. Let me know if you'd prefer the function export!